fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/TransactionManagerImpl.java

```diff
@@ -43,18 +50,28 @@ public synchronized Transaction create() {
         while(transactions.containsKey(txId)) {
             txId = randomUUID().toString();
         }
-        final Transaction tx = new TransactionImpl(txId);
+        final Transaction tx = new TransactionImpl(txId, this);
         transactions.put(txId, tx);
         return tx;
     }
 
     @Override
     public Transaction get(final String transactionId) {
         if(transactions.containsKey(transactionId)) {
-            return transactions.get(transactionId);
+            final Transaction transaction = transactions.get(transactionId);
+            if(transaction.hasExpired()) {
```

Comment:
please format with spaces after `if`'s in this file

Reply:
Still need to update this formatting. There are also a few other formatting issues in this file, like there should be a space between `while` and its opening `(` at line 53